docs/src/main/asciidoc/hibernate-search-orm-elasticsearch.adoc

```diff
@@ -1015,6 +1015,88 @@ You can enable AWS request signing in Hibernate Search by adding a dedicated ext
 See link:{hibernate-search-orm-elasticsearch-aws-guide}#aws-configuration-reference[the documentation for the Hibernate Search ORM + Elasticsearch AWS extension]
 for more information.
 
+[[management]]
+== Management endpoint
+
+:extension-status: preview
+include::{includes}/extension-status.adoc[]
+
+Hibernate Search extension has a capability to reindex the data through the xref:./management-interface-reference.adoc[management interface].
+By default, this endpoint is not available. It can be enabled through configuration properties as shown below.
+
+[source,properties]
+----
+quarkus.management.enabled=true <1>
+quarkus.hibernate-search.management.enabled=true <2>
+----
+<1> Enable the xref:./management-interface-reference.adoc[management interface].
+<2> Enable Hibernate Search specific management endpoints.
+
+Once the management is enabled, data can be re-indexed via `/q/reindex`, where `/q` is the default management root path.
+Reindex path (`/reindex`) can be changed via configuration property as shown below.
+
+[source,properties]
+----
+quarkus.hibernate-search.management.root-path=custom-root-path <1>
```

Comment:
Feels weird that `/reindex` is called the "root" path... it would mean that if we ever add another endpoint to the management interface, it can only be named `/reindex/something`?

Wouldn't it make sense for the default to be something like `/q/hibernate-search/reindex` and for the "root path" to only change "hibernate-search" (e.g. it'd become `/q/custom-root-path/reindex`)?